sheetstorm/tests/test_middleware.py

```diff
@@ -0,0 +1,26 @@
+from django.test import TestCase
+from django.urls import reverse
+
+from users.factories import UserFactory
+
+
+class ForceChangePasswordMiddlewareTests(TestCase):
+    def setUp(self):
+        self.url = reverse("custom-user-update")
+        self.user = UserFactory(need_change_password=True)
+
+    def test_user_should_not_access_website_if_not_authenticated(self):
+        response = self.client.get(self.url)
+        self.assertEqual(response.status_code, 302)
```

Comment:
You need to check what is a redirected url because status_code `302` doesn't give 100% sure that everything is working properly.

Reply:
Done